src/lib/provable/scalar-field.ts

```diff
@@ -0,0 +1,44 @@
+import { createField } from './core/field-constructor.js';
+import { createForeignField } from './foreign-field.js';
+import { field3ToShiftedScalar } from './gadgets/native-curve.js';
+import { Provable } from './provable.js';
+import { Scalar } from './scalar.js';
+
+export { ScalarField };
+
+/**
+ * ForeignField representing the scalar field of Pallas and the base field of Vesta
+ */
+class ScalarField extends createForeignField(
+  28948022309329048855892746252171976963363056481941647379679742748393362948097n
```

Comment:
can you import this constant? e.g. `Fq.modulus` where `Fq` is imported like in `scalar.ts`